templating/formats.rst

```diff
@@ -22,36 +22,68 @@ In many cases, you may want to allow a single controller to render multiple
 different formats based on the "request format". For that reason, a common
 pattern is to do the following::
 
-    public function indexAction(Request $request)
+    // ...
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+    class ArticleController extends Controller
     {
-        $format = $request->getRequestFormat();
+        /**
+         * @Route("/{slug}")
+         */
+        public function showAction(Request $request, Article $article)
```

Comment:
I'm a bit worried about the `Article $article` param here because it requires the extra bundle with the param converters. Do any of you see a problem with that too?